admin/css/boldgrid-backup-admin-premium.css

```diff
@@ -0,0 +1,106 @@
+/**
+ * Style for specific premium feature cards.
+ */
+
+#bgbkup_amazon_s3 .bglib-card-icon {
+	border-top-color: #feb424;
+}
+
+#bgbkup_dream_objects .bglib-card-icon {
+	border-top-color: #3e95be;
+}
+
+#bgbkup_backups .bglib-card-icon {
+	border-top-color: #0073aa;
+}
+
+#bgbkup_google_drive .bglib-card-icon {
+	border-top-color: #1ba160;
+}
+
+#bgbkup_amazon .bglib-card-icon {
+	border-top-color: #feb424;
+}
+
+#bgbkup_dream_objects .bglib-card-icon {
+	border-top-color: #3e95be;
+}
+
+#bgbkup_history .bglib-card-icon {
+	color: #f04207;
+	border-top-color: #f04207;
+}
+
+#bgbkup_historical_versions .bglib-card-icon {
+	color: #9ebaa0;
+	border-top-color: #9ebaa0;
+}
+
+#bgbkup_plugin_editor_tools .bglib-card-icon {
+	color: #5d824b;
+	border-top-color: #5d824b;
+}
+
+#bgbkup_backups .bglib-card-icon .dashicons {
+	color: #0073aa;
+}
+
+#bgbkup_updates .bglib-card-icon {
+	border-top-color: #aa0073;
+}
+
+#bgbkup_updates .bglib-card-icon .dashicons {
+	color: #aa0073;
+}
+
+#bgbkup_one_click_restoration .bglib-card-icon {
+	color: #c604c6;
+	border-top-color: #c604c6;
+}
+
+#bgbkup_get_premium .bglib-card-icon {
+	border-top-color: #f95b26;
+}
+
+#bgbkup_get_premium .bglib-card-icon .dashicons {
+	color: #f95b26;
+}
+
+/**
+ * Misc.
+ */
+
+/* The get premium button in the top notice. */
+.bglib-card {
```

Comment:
I don't think any of the classes below are needed.

Except for `.bg-box-title`. That can be moved to the library so other plugins can use it too.

Reply:
The changes I added to the CSS at the bottom of this were meant to be specific to the premium features page of the Total Upkeep plugin. If I made the changes in the css in the library, it would then affect other cards on other pages ( like the dashboard ).